<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CourierCompanyFormClass</class>
 <widget class="QMainWindow" name="CourierCompanyFormClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>255</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CourierCompanyForm</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="listWidgetPackages">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>321</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>234</width>
      <height>200</height>
     </rect>
    </property>
    <property name="title">
     <string>Add new package</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Recpient:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" colspan="2">
      <widget class="QLineEdit" name="lineEditRecipient"/>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Address Street:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="lineEditStreet"/>
     </item>
     <item row="2" column="0" colspan="3">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Address Number:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLineEdit" name="lineEditNumber"/>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Location X:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2" colspan="2">
      <widget class="QLineEdit" name="lineEditX"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Location Y:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="3">
      <widget class="QLineEdit" name="lineEditY"/>
     </item>
     <item row="5" column="0" colspan="4">
      <widget class="QPushButton" name="pushButtonAdd">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="CourierCompanyForm.qrc"/>
 </resources>
 <connections/>
</ui>
